nym-vpn-core/Cargo.toml

```diff
@@ -97,22 +99,22 @@ url = "2.5"
 vergen = { version = "8.3.1", default-features = false }
 zeroize = "1.6.0"
 
-nym-authenticator-requests = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-bandwidth-controller = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-bin-common = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-client-core = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-config = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-credential-storage = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-credentials = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-crypto = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-explorer-client = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-http-api-client = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-id = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-ip-packet-requests = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-node-requests = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-pemstore = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-sdk = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-task = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-topology = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-validator-client = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
-nym-wireguard-types = { git = "https://github.com/nymtech/nym", rev = "f86050d" }
+nym-authenticator-requests = { git = "https://github.com/nymtech/nym", rev = "ab7110586da46df7fef8ffa6b775c4d16b5ad35c" }
```

Comment:
I like the script and the abbreviated hashes it produces ;)

Reply:
I'm getting `sed: 1: "s/(nym-.* = \{ git = "h ...": RE error: invalid repetition count(s)` on MacOS. I'll try fixing it separately from this PR